fpdf/fpdf.py

```diff
@@ -4963,7 +4992,8 @@ def _is_svg(bytes):
     "XPos",
     "YPos",
     "get_page_format",
-    "ImageInfo",
```

Comment:
What will become of end-users code like this?
```python
from fpdf.fpdf import ImageInfo
```
I think this would break when the next release will be published...

Maybe we should keep `"ImageInfo"` there,
just for the sake of backward-compatibility,
even if it's not used in this module?

Reply:
We can keep it there, no problem.